cornflow/models/meta_model.py

```diff
@@ -129,7 +141,7 @@ def get_all_objects(
         """
         query = cls.query.filter(cls.deleted_at == None)
         # TODO: in airflow they use: query = session.query(ExecutionModel)
-        if not user.is_admin():
+        if not user.is_admin() and not user.is_service_user():
```

Comment:
why does `is_admin()` returns True for the service user? `is_admin` should check for admin role exclusively, right?

Reply:
is_admin should only return True for admin, not for service user.